<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetItemView</class>
 <widget class="QDialog" name="SetItemView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox{
	border-style:solid;
	border:1px solid black;
}
QPushButton{
	border-style:solid;
	border-radius:3px;
}
QPushButton:hover{
	background-color:red;
}</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>0</y>
     <width>311</width>
     <height>291</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>281</width>
       <height>231</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style:solid;
border:1px solid black</string>
     </property>
     <property name="title">
      <string>心音模拟听诊模式选择</string>
     </property>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style:none</string>
      </property>
      <property name="text">
       <string>肺部区域听诊时 无呼吸音</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>150</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style:none</string>
      </property>
      <property name="text">
       <string>肺部区域听诊时 有呼吸音</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>291</width>
       <height>221</height>
      </rect>
     </property>
     <property name="title">
      <string>心肺音扩音听诊模式选择</string>
     </property>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>心肺音直接扩音听诊</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>231</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>电子听诊器接触式扩音听诊</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style:solid;
border:1px solid black;
font-size:20px;</string>
     </property>
     <property name="value">
      <number>18</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:16px;</string>
     </property>
     <property name="text">
      <string>实验台数：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:16px;</string>
     </property>
     <property name="text">
      <string>设置WebIP:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="web_ip">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>240</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style:solid;
border:1px solid black;
font-size:18px;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>120</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="btn1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>75</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>75</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>75</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="la1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>心音模拟听诊</string>
    </property>
   </widget>
   <widget class="QLabel" name="la2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>心肺扩音听诊</string>
    </property>
   </widget>
   <widget class="QLabel" name="la3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>系统设置</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>310</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
